examples/low-level-api/create_vp.rs

```diff
@@ -51,62 +57,107 @@ pub async fn create_vp() -> Result<(Presentation, IotaDocument, IotaDocument)> {
     .credential(credential)
     .build()?;
 
-  // Sign the presentation with the holders private key.
-  //
-  // Optionally include a challenge from the requester in the signature.
+  // We now sign the presentation with the holder's private key and include a challenge and an expiry timestamp 10
+  // minutes from now.
   // A unique random challenge generated by the requester per presentation can mitigate replay attacks
-  // (along with other properties like `expires` and `domain`).
+  // (along with other properties like `expires` and `domain`). The expiry timestamp enables the
+  // verifier to drop the challenge from memory after a specified amount of time has passed.
```

Comment:
I see your confusion. It will not be automatically dropped it will just fail validation. My point was just that it can be nice to use them together if one has tight memory constraints, but still prefers the absolute mitigation that "challenge" gives. After the specified time has passed it means you will no longer accept the credential hence you can drop the challenge. Don't know if this will be a common use case however so maybe it is better to just use one of the two. On the other hand it is nice to illustrate more of what one can do without needing to make too many examples.

Reply:
I see, thanks for the explanation. I'm not sure how much of a memory concern the challenge really is given its small size, so this feels more like a niche concern to me and thus not all that necessary for an example. I'm fine either way, though.

Reply:
Ok in that case I will just remove the use of `expires`.

Reply:
Ah, what I actually meant was just the comment mentioning how `expires` can be used, not the use of `expires` itself, which seems fine to me. Sorry for the confusion.